### PR TITLE
Wownero Integration

### PR DESCRIPTION
**Pull request by OmarHatem28**
Issue Number (if Applicable): Fixes #

# Description

Please include a summary of the changes and which issue is fixed / feature is added. 

# Pull Request - Checklist  

- [ ] Initial Manual Tests Passed
- [ ] Double check modified code and verify it with the feature/task requirements
- [ ] Format code
- [ ] Look for code duplication
- [ ] Clear naming for variables and methods

### CODE REVIEW

**Comment by nahuhh** [comment]
Thanks Omar!

File: .github/workflows/pr_test_build.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
           channel: stable
 
       - name: Install package dependencies
-        run: sudo apt-get install -y curl unzip automake build-essential file pkg-config git python libtool libtinfo5 cmake clang
+        run: sudo apt-get install -y curl unzip automake build-essential file pkg-config git python libtool libtinfo5 cmake clang libzmq3-dev zmq
```
Comment (nahuhh) [inline .github/workflows/pr_test_build.yml:49]:
```suggestion
        run: sudo apt-get install -y curl unzip automake build-essential file pkg-config git python libtool libtinfo5 cmake clang libzmq3-dev
```

i dont think there is a `zmq` package for ubuntu 20.04

The actions build fails with libwallet stuff being unavail. I'm not sure that is zmq related

Comment (OmarHatem28) [reply .github/workflows/pr_test_build.yml]: yeah I noticed, but there are a lot of problems, so I am trying to see which one of them is the cause of failure. the first error was `Could not find required header zmq.h` so I added `libzmq3-dev`
but then it followed with the second error `Could not find required libzmq`
now I am thinking that this was not the issue, rather this error is the issue
```
fatal: remote error: upload-pack: not our ref 6405ac6e***708938ffd8631***9e187879bf74d***f3
2024-03-22T11:18:38.1938071Z fatal: Fetched in submodule path 'external/trezor-common', but it did not contain 6405ac6e***708938ffd8631***9e187879bf74d***f3. Direct fetching of that commit failed.
```

Comment (OmarHatem28) [reply .github/workflows/pr_test_build.yml]: but at least there are some errors here that I can trace, meanwhile on iOS, it just fails 😭

**Comment by nahuhh** [comment]
why only 12 threads?! #wow 🥳🥳😆

**Comment by OmarHatem28** [comment]
finally :3 

Screenshot 2024-03-24 at 5 33 24 AM


Screenshot 2024-03-24 at 5 31 02 AM


will check what's missing soon and then will try to enable it for iOS as well

**Comment by nahuhh** [comment]
- [ ] shows msg to backup 24 words
- [ ] still shows option for legacy seed generation, but doesnt allow legacy seed restore. (should be other way around - accept legacy or polyseed for restore, but only generate polyseeds)
- [ ] doesnt seem to select a node by default
- [ ] its not syncing for me 

@sanderfoobar the apk i tested is on the actions build

**Comment by OmarHatem28** [comment]
@sanderfoobar yeah sure ^^
in the meantime, if you can check the synchronization issue that would be great

**Comment by MrCyjaneK** [comment]
@OmarHatem28 the synchronization issue may be the overlapping symbols issue, to verify - disable monero plugin entirely and use wownero

**Comment by OmarHatem28** [comment]
@sanderfoobar nope, nothing so far, we are just holding this off for a bit as we are getting close to releasing Hardware Wallets and Bitcoin's Silent Payments, so we are focusing on that currently, but I will keep you updated on this as soon as I can

**Comment by nahuhh** [comment]
@OmarHatem28 can you edit or redo this pr (or cherrypick) using the branch that @sanderfoobar posted?

im excited to see it sync

**Comment by OmarHatem28** [comment]
closed in favor of #1485

**Pull request closed without merging**